package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fitbit/sdk",
-  "version": "6.1.0-pre.2",
+  "version": "6.0.0-pre.2",
```

Comment:
NIT: "6.0.1-pre.0"?

Reply:
Sorry, I should have added a TODO, this is just set back to 6.0 temporarily to enable testing, since I enabled this polyfill only on SDK <6.1.0.